Content.Shared/_RMC14/Entrenching/BarricadeSystem.cs

```diff
@@ -100,6 +101,9 @@ private void OnSandbagDismantleDoAfter(Entity<EntrenchingToolComponent> tool, re
         if (TryComp(args.Target, out DamageableComponent? damageable))
             bagsSalvaged -= Math.Max((int) damageable.TotalDamage / barricade.MaterialLossDamageInterval - 1, 0);
 
+        if (TryComp(args.Target, out BarbedComponent? barbed) && barbed.IsBarbed)
+            EntityManager.SpawnEntity(barbed.Spawn, EntityManager.GetCoordinates(args.Coordinates));
```

Comment:
This can use shorter methods since this code is in an entity system	
```suggestion
            Spawn(barbed.Spawn, GetCoordinates(args.Coordinates));
```

Reply:
Got rid of the other instance that was there too